### ISSUE TITLE
[BUG] "ERROR: Unrecognized pair style 'deepmd' " while using dpgen in the model_devi stage

### ISSUE THREAD

**Issue by user-ting** [deepmodeling/deepmd-kit#3587]
### Bug summary

I installed deepmd-kit 2.2.7 with conda:" conda create -n deepmd deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps -c https://conda.deepmodeling.com -c defaults", (somehow the dpgen and dpdata were also installed into the deepmd enviroment). While i was using dpgen, in the model_devi stage, all jobs terminated again and again with "ERROR: Unrecognized pair style 'deepmd'" in lammps output file. While i was submitting the dpgen job with "dpgen run para.jason machine.jason", the deepmd enviroment was already activated, and the "echo $LAMMPS_PLUGIN_PATH" gave "/home1/zhangzt/anaconda3/envs/deepmd/lib/deepmd_lmp", so i don't have any other ideas what caused the error.

### DeePMD-kit Version

DeePMD-kit v2.2.7

### TensorFlow Version

the train.log file gave"WARNING:tensorflow:From /home1/zhangzt/anaconda3/envs/deepmd/lib/python3.10/site-packages/tensorflow/python/compat/v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version."

### How did you download the software?

conda

### Input Files, Running Commands, Error Log, etc.

para.jason:
{
  "type_map": [
    "C"
  ],
  "mass_map": [
    12
  ],
  "init_data_prefix": "/home1/zhangzt/mlPotential_DP/dpgen_test_new/init/",
  "init_data_sys": [
    "density_varying_init/1.0_5000K/deepmd_data",
    "density_varying_init/1.6_5000K/deepmd_data",
    "density_varying_init/2.2_5000K/deepmd_data",
    "density_varying_init/2.8_5000K/deepmd_data",
    "density_varying_init/3.0_5000K/deepmd_data",
    "density_varying_init/3.0_9000K/deepmd_data",
    "density_varying_init/3.2_5000K/deepmd_data",
    "density_varying_init/3.2_9000K/deepmd_data",
    "density_varying_init/3.3_5000K/deepmd_data",
    "density_varying_init/3.4_5000K/deepmd_data",
    "density_varying_init/3.4_9000K/deepmd_data",
    "density_varying_init/3.5_5000K/deepmd_data",
    "density_varying_init/3.6_5000K/deepmd_data",
    "density_varying_init/3.6_9000K/deepmd_data"
  ],
  "sys_configs": [
    ["../mdconfigs/dens_1.0/00*/POSCAR"],
    ["../mdconfigs/dens_1.2/00*/POSCAR"],
    ["../mdconfigs/dens_1.4/00*/POSCAR"],
    ["../mdconfigs/dens_1.6/00*/POSCAR"],
    ["../mdconfigs/dens_1.8/00*/POSCAR"],
    ["../mdconfigs/dens_2.0/00*/POSCAR"],
    ["../mdconfigs/dens_2.2/00*/POSCAR"],
    ["../mdconfigs/dens_2.4/00*/POSCAR"],
    ["../mdconfigs/dens_2.6/00*/POSCAR"],
    ["../mdconfigs/dens_2.8/00*/POSCAR"],
    ["../mdconfigs/dens_2.9/00*/POSCAR"],
    ["../mdconfigs/dens_3.0/00*/POSCAR"],
    ["../mdconfigs/dens_3.1/00*/POSCAR"],
    ["../mdconfigs/dens_3.2/00*/POSCAR"],
    ["../mdconfigs/dens_3.3/00*/POSCAR"],
    ["../mdconfigs/dens_3.4/00*/POSCAR"],
    ["../mdconfigs/dens_3.5/00*/POSCAR"],
    ["../mdconfigs/dens_3.6/00*/POSCAR"],
    ["../mdconfigs/dens_4.0/00*/POSCAR"],
    ["../mdconfigs/dens_4.4/00*/POSCAR"],
    ["../mdconfigs/dens_4.8/00*/POSCAR"],
    ["../mdconfigs/stru_mono/000/POSCAR"],
    ["../mdconfigs/stru_surf/000/POSCAR"],
    ["../mdconfigs/001/POSCAR"]
  ],
  "numb_models": 4,
  "default_training_param": {
    "model": {
      "_comment": " model parameters",
      "type_map": [
        "C"
      ],
      "descriptor": {
        "type": "se_e3",
        "sel": [
          150
        ],
        "rcut_smth": 0.5,
        "rcut": 5.0,
        "neuron": [
          20,
          40,
          80
        ],
        "resnet_dt": false,
        "seed": 17
      },
      "fitting_net": {
        "type": "ener",
        "neuron": [
          60,
          60,
          60
        ],
        "resnet_dt": false,
        "seed": 9
      }
    },
    "learning_rate": {
      "type": "exp",
      "start_lr": 0.001,
      "stop_lr": 1e-08,
      "decay_steps": 1000
    },
    "loss": {
      "type": "ener",
      "start_pref_e": 0.02,
      "limit_pref_e": 2,
      "start_pref_f": 1000,
      "limit_pref_f": 1,
      "start_pref_v": 0,
      "limit_pref_v": 0
    },
    "training": {
      "numb_steps": 10000,
      "seed": 11,
      "disp_file": "lcurve.out",
      "disp_freq": 1,
      "save_freq": 40,
      "save_ckpt": "model.ckpt",
      "disp_training": true,
      "time_training": true,
      "profiling": false,
      "profiling_file": "timeline.json"
    }
  },
  "fp_task_max": 3000,
  "fp_task_min": 5,
  "model_devi_engine": "lammps",
  "model_devi_jobs": [
    {
      "sys_idx": [
        0,1,2,3,4,5,6,7,8,
        9,10,11,12,13,14,15,16,
        17,18,19,20,21,22
      ],
      "temps": [
        300,
        1000,
        5000,
        9000
      ],
      "press": [
        1,
        50000,
        500000,
        5000000
      ],
      "trj_freq": 100,
      "nsteps": 10000,
      "ensemble": "npt"
    }
  ],
  "model_devi_dt": 0.002,
  "model_devi_skip": 0,
  "model_devi_f_trust_lo": 0.01,
  "model_devi_f_trust_hi": 2,
  "model_devi_clean_traj": false,
  "shuffle_poscar": false,
  "fp_style": "vasp",
  "fp_pp_path": "../ff",
  "fp_pp_files": [
    "POTCAR"
  ],
  "fp_incar": "../incar/INCAR"
}

-----------------------------------
machine.jason:
{
  "api_version": "1.0",
  "deepmd_version": "2.2.7",
  "train": {
    "command": "/home1/zhangzt/anaconda3/envs/deepmd/bin/dp",
    "machine": {
      "batch_type": "LSF",
      "local_root": "./",
      "remote_root": "/home1/zhangzt/mlPotential_DP/dpgen_test_new/work",
      "context_type": "LocalContext"

    },
    "resources": {
      "number_node": 1,
      "cpu_per_node": 24,
      "queue_name": "E5-2680",
      "group_size": 1,
      "para_deg": 4,
      "batch_type": "LSF",
      "kwargs": {
        "gpu_usage": false
      }
    }
  },
  "model_devi": {
    "command": "/home1/zhangzt/anaconda3/envs/deepmd/bin/lmp",
    "machine": {
      "batch_type": "LSF",
      "local_root": "./",
      "remote_root": "/home1/zhangzt/mlPotential_DP/dpgen_test_new/work",
      "context_type": "LocalContext"

    },
    "resources": {
      "number_node": 1,
      "cpu_per_node": 12,
      "queue_name": "E5-2630",
      "group_size": 1,
      "para_deg": 9,
      "batch_type": "LSF",
      "kwargs": {
        "gpu_usage": false
      }
    }
  },
  "fp": {
    "command": "/home1/zhangzt/mylibs/vasp/vasp.5.4.4/bin/vasp_gam",
    "machine": {
      "batch_type": "LSF",
      "local_root": "./",
      "remote_root": "/home1/zhangzt/mlPotential_DP/dpgen_test_new/work",
      "context_type": "LocalContext"

    },
    "resources": {
      "number_node": 1,
      "cpu_per_node": 24,
      "queue_name": "E5-2680",
      "group_size": 1,
      "para_deg": 4,
      "batch_type": "LSF",
      "kwargs": {
        "gpu_usage": false
      }
    }
  }
}

-----------------------------------
error log files was lost, though

### Steps to Reproduce

1: conda create -n deepmd deepmd-kit=*=*cpu libdeepmd=*=*cpu lammps -c https://conda.deepmodeling.com -c defaults

2: conda activate deepmd

2: dpgen run carbon.jason machine.jason

**Comment by njzjz** [comment]
Dpdispatcher launches the submission scripts with `bash -l`, [initializing the shell from your `.bashrc` file](https://stackoverflow.com/a/20499839/9567349). Thus, it's required to set the environment explicitly in the input file.
